boards.txt

```diff
@@ -818,3 +818,50 @@ ng_beacon.menu.softdevice.s130.softdeviceversion=2.0.1
 ng_beacon.menu.softdevice.s130.upload.maximum_size=151552
 ng_beacon.menu.softdevice.s130.build.extra_flags=-DNRF51 -DS130 -DNRF51_S130
 ng_beacon.menu.softdevice.s130.build.ldscript=armgcc_s130_nrf51822_xxaa.ld
+
+
+Sinobit.name=Sino:bit
+
+Sinobit.vid.0=0x0d28
+Sinobit.pid.0=0x0204
+
+Sinobit.upload.tool=sandeepmistry:openocd
+Sinobit.upload.protocol=cmsis-dap
+Sinobit.upload.target=nrf51
+Sinobit.upload.maximum_size=262144
+Sinobit.upload.setup_command=transport select swd;
+Sinobit.upload.use_1200bps_touch=false
+Sinobit.upload.wait_for_upload_port=false
+Sinobit.upload.native_usb=false
+
+Sinobit.bootloader.tool=sandeepmistry:openocd
+
+Sinobit.build.mcu=cortex-m0
+Sinobit.build.f_cpu=16000000
+Sinobit.build.board=BBC_MICROBIT
```

Comment:
This should be updated as well.

Reply:
Is that a compiler parameter or something i can set to any arbitrary string?

Reply:
Compiler parameter, would get mapped to `ARDUINO_<value>` as a compile time define (`-DARDUINO_<value>`).